### ISSUE TITLE
Nature Reviews ... problem

### ISSUE THREAD

**Issue by sckott** [ourresearch/journalsdb#25]
Hi Casey, A user reported an issue with their Nature subscriptions in Unsub where some titles weren't showing up that they had uploaded title prices for. I had a look and there were 14 titles that all had titles as `Nature Reviews` - BUT they should have been `Nature Reviews Neuroscience`, `Nature Reviews Cancer`, etc. The other issue was that some of these Nature Reviews XXX titles also had the publisher as `null` in journalsdb. 

Here's the ones where title is wrong and publisher is null

- https://api.journalsdb.org/journals/1471-003X but https://api.crossref.org/journals/1471-0048 (nature reviews neuroscience)
- https://api.journalsdb.org/journals/1474-175X but https://api.crossref.org/journals/1474-1768 (nature reviews cancer)
- https://api.journalsdb.org/journals/1474-1733 but https://api.crossref.org/journals/1474-1741 (nature reviews immunology)

Here's the ones where just title is wrong. I only checked some of the below ISSN's with the journalsdb API, and for those that I checked the title was correct, but somehow we ended up with just `Nature reviews` in Unsub. Maybe the titles were fixed recently? 

1740-1526 | Nature reviews
1759-5061 | Nature reviews
1759-5002 | Nature reviews
1759-4774 | Nature reviews
1759-5045 | Nature reviews
1471-0056 | Nature reviews
1759-4758 | Nature reviews
1759-5029 | Nature reviews
1759-4790 | Nature reviews
1471-0072 | Nature reviews
1474-1776 | Nature reviews

Heather says to say this is a high priority fix

thanks! and could you let me know when these are fixed?

**Comment by caseydm** [comment]
Hi Scott this is fixed in journalsdb. I updated titles and publisher for those three that were incorrect. I also ran a query to see if there were other issues and found one minor. Nature Reviews Materials was listed as "Nature Reviews. Materials" so I removed the period. See attached for the list of Nature Reviews titles in the database.

As for getting the correct titles in unsub, I am showing those titles were updated in journalsdb around July 30th. As you said, the titles were likely identified during a QA process and updated to the correct ones back then. The only way I know titles were possibly pulled from JournalsDB is through a xplenty job which is possibly called "journalsdb to redshift". But that job is no longer active and was last ran July 19th. So maybe that is why the new titles are not pulled in? I would have to defer to Heather on that.

Attachment:
<img width="758" alt="Screen Shot 2021-10-06 at 1 00 13 PM" src="https://user-images.githubusercontent.com/3642984/136258200-c61e8597-1e18-4982-8f2e-a9e719b39879.png">
